frontend2/src/components/sidebar/__test__/Sidebar.test.tsx

```diff
@@ -45,6 +45,8 @@ test("UI: should collapse sidebar", () => {
             leaveMyTeam: async (): Promise<void> => {
               await Promise.resolve();
             },
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
+            refreshTeam: (team): void => {},
```

Comment:
Ope I think you have this in two PRs haha... I would make it so it's only in one to save yourself a headache 😆